week02/Py7.py

```diff
@@ -0,0 +1,6 @@
+def even_index(a):
+    evenindex=a[::2]
+    print(' '.join([str(elem) for elem in evenindex]))
```

Comment:
чтобы применить функцию ко всем элементам списка есть map, для названия переменных используй snake_case
```suggestion
    print(' '.join(map(str, even_index)))
```